CHANGELOG.md

```diff
@@ -1,3 +1,304 @@
+
+*new* enum value: `MDCBannerViewLayoutStyleSingleRow` in `MDCBannerViewLayoutStyle`
+
+*new* enum: `MDCBannerViewLayoutStyle`
+
+*new* typedef: `MDCBannerViewLayoutStyle`
+
+#### MDCBannerView
+
+*new* property: `bannerViewLayoutStyle` in `MDCBannerView`
+
+*new* property: `showsDivider` in `MDCBannerView`
+
+*new* property: `dividerColor` in `MDCBannerView`
+
+### BottomNavigation
+
+#### MDCBottomNavigationBar(ToBeDeprecated)
+
+*new* category: `MDCBottomNavigationBar(ToBeDeprecated)`
+
+*modified* property: `sizeThatFitsIncludesSafeArea` in `MDCBottomNavigationBar(ToBeDeprecated)`
+
+| Type of change: | parent.usr |
+|---|---|
+| From: | `c:objc(cs)MDCBottomNavigationBar` |
+| To: | `c:objc(cy)MDCBottomNavigationBar@ToBeDeprecated` |
+
+#### MDCBottomNavigationBar
+
+*new* property: `enableRippleBehavior` in `MDCBottomNavigationBar`
+
+### ButtonBar
+
+#### MDCButtonBar
+
+*new* property: `enableRippleBehavior` in `MDCButtonBar`
+
+*new* property: `rippleColor` in `MDCButtonBar`
+
+#### MDCButtonBar(ToBeDeprecated)
+
+*new* category: `MDCButtonBar(ToBeDeprecated)`
+
+*modified* property: `inkColor` in `MDCButtonBar(ToBeDeprecated)`
+
+| Type of change: | parent.usr |
+|---|---|
+| From: | `c:objc(cs)MDCButtonBar` |
+| To: | `c:objc(cy)MDCButtonBar@ToBeDeprecated` |
+
+### Dialogs
+
+#### MDCAlertControllerView
+
+*new* property: `enableRippleBehavior` in `MDCAlertControllerView`
+
+#### MDCAlertController
+
+*new* property: `enableRippleBehavior` in `MDCAlertController`
+
+### List
+
+#### MDCBaseCell
+
+*new* property: `rippleColor` in `MDCBaseCell`
+
+*new* property: `enableRippleBehavior` in `MDCBaseCell`
+
+#### MDCBaseCell(ToBeDeprecated)
+
+*new* category: `MDCBaseCell(ToBeDeprecated)`
+
+*modified* property: `inkColor` in `MDCBaseCell(ToBeDeprecated)`
+
+| Type of change: | parent.usr |
+|---|---|
+| From: | `c:objc(cs)MDCBaseCell` |
+| To: | `c:objc(cy)MDCBaseCell@ToBeDeprecated` |
+
+### NavigationBar
+
+#### MDCNavigationBar
+
+*new* property: `enableRippleBehavior` in `MDCNavigationBar`
+
+*new* property: `rippleColor` in `MDCNavigationBar`
+
+#### MDCNavigationBar(ToBeDeprecated)
+
+*new* category: `MDCNavigationBar(ToBeDeprecated)`
+
+*modified* property: `inkColor` in `MDCNavigationBar(ToBeDeprecated)`
+
+| Type of change: | parent.usr |
+|---|---|
+| From: | `c:objc(cs)MDCNavigationBar` |
+| To: | `c:objc(cy)MDCNavigationBar@ToBeDeprecated` |
+
+### Slider
+
+#### MDCSlider(ToBeDeprecated)
+
+*new* category: `MDCSlider(ToBeDeprecated)`
+
+*modified* property: `inkColor` in `MDCSlider(ToBeDeprecated)`
+
+| Type of change: | parent.usr |
+|---|---|
+| From: | `c:objc(cs)MDCSlider` |
+| To: | `c:objc(cy)MDCSlider@ToBeDeprecated` |
+
+#### MDCSlider
+
+*new* property: `enableRippleBehavior` in `MDCSlider`
+
+*new* property: `rippleColor` in `MDCSlider`
+
+### Snackbar
+
+#### MDCSnackbarMessage
+
+*new* property: `enableRippleBehavior` in `MDCSnackbarMessage`
```

Comment:
Should `rippleColor` be here?

Reply:
i only added rippleColor where inkColor existed prior